deltachat-ios/Controller/NewChatViewController.swift

```diff
@@ -6,10 +6,11 @@ class NewChatViewController: UITableViewController {
     private let dcContext: DcContext
 
     private let sectionNew = 0
-    private let sectionNewRowNewContact = 0
+    private let sectionNewScanQRCode = 0
     private let sectionNewRowNewGroup = 1
     private let sectionNewRowBroadcastList = 2
-    private var sectionNewRowCount: Int { return UserDefaults.standard.bool(forKey: "broadcast_lists") ? 3 : 2 }
+    private let sectionNewRowNewContact = 3
+    private var sectionNewRowCount: Int { return UserDefaults.standard.bool(forKey: "broadcast_lists") ? 4 : 3 }
```

Comment:
i've seen this nicer for other ViewControllers, but i do not want to do a refactoring for this small change :)
also, broadcast lists will  be non-experimental soon(tm) anyway, then it does not look _that_ hacky :)